src/pages/StudyManage/index.tsx

```diff
@@ -0,0 +1,58 @@
+import { useEffect, useState } from 'react';
+import {
+  StudyManageButtonWrapper,
+  StudyManageContainer,
+} from '@src/pages/StudyManage/style';
+import { studyManageType } from '@src/interfaces/studyManage';
+import { StudyManageMemberControl } from '@src/containers';
+import { getStudyManageInfomation } from '@src/apis/studyManage';
+import { Button, Typography } from '@mui/material';
+
+function StudyManage() {
+  const [data, setData] = useState({} as studyManageType);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const res = await getStudyManageInfomation();
+      setData(res);
+    };
+
+    fetchData();
+  }, []);
+
+  const getTopic = () => {
+    const { topic = '' } = data;
+    return topic;
+  };
+
+  const getTitle = () => {
+    const { title = '' } = data;
+    return title;
+  };
+
+  return (
+    <StudyManageContainer>
+      <div>
+        <Typography variant='subtitle2' color='secondary'>
+          {getTopic()}
+        </Typography>
+        <Typography variant='h4'>{getTitle()}</Typography>
```

Comment:
여기도 확인해보니 폰트가 조금 다른 것 같습니다!

```suggestion
        <Typography variant='body1' color='secondary'>
          {getTopic()}
        </Typography>
        <Typography variant='h3'>{getTitle()}</Typography>
```